packages/base/cards-grid.gts

```diff
@@ -226,6 +226,20 @@ class Isolated extends Component<typeof CardsGrid> {
     if (!this.liveQuery) {
       return;
     }
+
+    // This is a nice place to measure the end of the app boot as this is where
+    // we'll get the instances to show in the index card's grid which is usually
+    // what we present when showing the app for the first time
+    if (
+      (globalThis as any).__bootStart &&
+      this.liveQuery.instances.length > 0
+    ) {
+      console.log(
+        `time since app boot: ${
+          performance.now() - (globalThis as any).__bootStart
+        } ms`,
+      );
```

Comment:
Is there a way to make this not show up in CI logs?

<img width="614" alt="boxel@59740e3 2024-06-03 14-32-40" src="https://github.com/cardstack/boxel/assets/43280/869d1cc0-9eea-49dd-857a-6e91c3624fe6">

Reply:
Yeah, I think there is a CI env var that we can use